### DISCUSSION TITLE
When importing Trading212 interests, a random name is given to the transactions flooding hte holdings page

### DISCUSSION THREAD

**Discussion by GerardPolloRebozado** [dickwolff/Export-To-Ghostfolio#73] [General]
Wouldn't be better to group them by currency?
![image](https://github.com/dickwolff/Export-To-Ghostfolio/assets/37554513/869bbd1a-543f-466b-8c8d-e19c89fc42b7)

**Comment by dickwolff** [comment]
Thanks for reporting this! The tool however hun hdoes not assign a random value. It takes the value from the `notes` field in the CSV export. I think in your example the transaction reference ID it put in that field for some reason.

Trading 212 is notorious for changing the export layout depending on which data you want to export. Did you perhaps select to only export interest? Could you maybe provide me with a (partial) CSV export so I can take a look at this in more detail?

**Comment by GerardPolloRebozado** [comment]
I tried exporting with only interest and tried with exporting everything but I get same results here you have an export with all transaction types

[from_2024-05-11_to_2024-05-13_MTcxNTYwMzEyNjM0OA.csv](https://github.com/dickwolff/Export-To-Ghostfolio/files/15294472/from_2024-05-11_to_2024-05-13_MTcxNTYwMzEyNjM0OA.csv)

**Comment by dickwolff** [comment]
Weirly enough, when I run and use your example file as input, I get the following results:

```
{
    "meta":{
       "date":"2024-05-13T12:35:26.247Z",
       "version":"v0"
    },
    "activities":[
       {
          "accountId":"2918fee8-180a-470c-9ac6-8138837d5fc7",
          "comment":"",
          "fee":0.06,
          "quantity":1,
          "type":"INTEREST",
          "unitPrice":0.06,
          "currency":"EUR",
          "dataSource":"MANUAL",
          "date":"2024-05-11T21:02:44+02:00",
          "symbol":"Interest on cash"
       },
       {
          "accountId":"2918fee8-180a-470c-9ac6-8138837d5fc7",
          "comment":"",
          "fee":0.05,
          "quantity":1,
          "type":"INTEREST",
          "unitPrice":0.05,
          "currency":"EUR",
          "dataSource":"MANUAL",
          "date":"2024-05-12T21:02:49+02:00",
          "symbol":"Interest on cash"
       }
    ]
 }
```

![image](https://github.com/dickwolff/Export-To-Ghostfolio/assets/5620002/ae8b3377-9faf-4c3d-a126-394c64bdd56f)

This looks normal to me. Which version of the tool are you running currently?

**Reply by GerardPolloRebozado** [reply]
Using the same CSV I posted here it shows with correct name:
![image](https://github.com/dickwolff/Export-To-Ghostfolio/assets/37554513/16c5e454-9b8c-4854-ade9-59e8550ed942)
Then if you check what imported:
![image](https://github.com/dickwolff/Export-To-Ghostfolio/assets/37554513/91eba937-d0a7-4fc3-befd-c0b2ce10e05d)
Im using the version: 0.13.0

**Reply by dickwolff** [reply]
I have a repro on this issue. Will look into this more and try and figure out why this is happening.

![image](https://github.com/dickwolff/Export-To-Ghostfolio/assets/5620002/5903fe23-492c-4319-ada9-0a67d6e8747a)

**Comment by dickwolff** [comment]
I think this is a Ghostfolio related issue. I posted it [there](https://github.com/ghostfolio/ghostfolio/issues/3381) and will track the issue.
